config/initializers/honeybadger.rb

```diff
@@ -1,25 +1,26 @@
 # Can be used to implement more programatic error handling
 # https://docs.honeybadger.io/lib/ruby/getting-started/ignoring-errors.html#ignore-programmatically
 
-MESSAGE_FINGERPRINTS = {
-  "SuspendedError" => "banned",
-  "Rack::Timeout::RequestTimeoutException" => "rack_timeout",
-  "Rack::Timeout::RequestTimeoutError" => "rack_timeout",
-  "PG::QueryCanceled" => "pg_query_canceled"
-}.freeze
+# rubocop:disable Metrics/BlockLength
+Rails.application.reloader.to_prepare do
+  message_fingerprints = {
+    "SuspendedError" => "banned",
+    "Rack::Timeout::RequestTimeoutException" => "rack_timeout",
+    "Rack::Timeout::RequestTimeoutError" => "rack_timeout",
+    "PG::QueryCanceled" => "pg_query_canceled"
+  }
 
-COMPONENT_FINGERPRINTS = {
-  "internal" => "internal"
-}.freeze
+  component_fingerprints = {
+    "internal" => "internal"
+  }
 
-HONEYBADGER_EXCEPTIONS_TO_IGNORE = [
-  ActiveRecord::QueryCanceled,
-  ActiveRecord::RecordNotFound,
-  Pundit::NotAuthorizedError,
-  RateLimitChecker::LimitReached,
-].freeze
+  honeybadger_exceptions_to_ignore = [
+    ActiveRecord::QueryCanceled,
+    ActiveRecord::RecordNotFound,
+    Pundit::NotAuthorizedError,
+    RateLimitChecker::LimitReached,
+  ]
```

Comment:
these shouldn't be constants, they're only used here and they actively create problems when in development Rails tries to reload these, finding them already defined